k2/csrc/ragged_test.cu

```diff
@@ -128,6 +128,23 @@ TEST(RaggedTest, TestRaggedFromString) {
   }
 }
 
+TEST(RaggedTest, TestRaggedFromVecs) {
+  std::vector<std::vector<int32_t>> vecs;
+  vecs.emplace_back(std::vector<int32_t>{7, 9});
+  vecs.emplace_back(std::vector<int32_t>{10,12,13});
+  vecs.emplace_back(std::vector<int32_t>{});
+  Ragged<int32_t> r(vecs);
+  Array1<int32_t> row_splits(GetCpuContext(), std::vector<int32_t>{0, 2, 5, 5});
```

Comment:
You can write std::vector<int32_t> expected_row_splits = {0,2,5,5}`, then call `CheckArrayData(r.RowSplits(1), expected_row_splits).`